_content/monographs/book- of-verses-elder-bhikkhunis_bhikkhu-mahinda.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Therīgāthāpāli: Book of Verses of Elder Bhikkhunis"
+authors:
+  - "Bhikkhu Mahinda"
+external_url: "https://store.pariyatti.org/therigathapali-ebook"
+course: tg
+tags:
+  - pali-canon
+  - poetry
+year: 2017
+publisher:pariyatti 
```

Comment:
Must be a space between the colon and the value.  There must be no spaces in the file name.